modules/consul/consul.go

```diff
@@ -14,7 +14,7 @@ const (
 )
 
 const (
-	DefaultBaseImage = "docker.io/hashicorp/consul:1.15"
+	DefaultBaseImage = "docker.io/hashicorp/consul:1.18"
```

Comment:
I also updated the version. when this change was made it ported the same version of consul used in the example, but it isn't the current one.

Reply:
I'll give you some historical context about this: we usually define a base image as a reference, but we encourage always passing the `WithImage` option to make it clear what image to use: our examples in the website always include the image. So updating the default base image is just a matter of checking that the code of the module behaves as expected for that image. Like a minimal supported version. For this particular case, it will work because there is no specific logic, but if you check Elasticsearch module, it has custom code to detect versions and perform tasks according to that version. Therefore a simple bump in the default image could lead to changes in the implementation of the module.

But because we are not doing/building a support matrix for any module, it's fine to keep the default image as is, and probably change the code examples, only.

We are having conversations internally on how to address this: testcontainers-java forces passing the image as part of the "RunContainer" method, which is something we could do. But it's not decided yet.

Reply:
ok maybe it is best to just keep it coherent as it was vs different versions here and there. I'll revert this part. thanks for the context!